src/main/kotlin/com/dansplugins/factionsystem/MedievalFactions.kt

```diff
@@ -238,6 +232,64 @@ class MedievalFactions : JavaPlugin() {
         )
         setupRpkLockService()
 
+        val metrics = Metrics(this, 8929)
+        metrics.addCustomChart(
+            SimplePie("language_used") {
+                config.getString("language")
+            }
+        )
+        metrics.addCustomChart(
+            SimplePie("database_dialect") {
+                config.getString("database.dialect")
+            }
+        )
+        metrics.addCustomChart(
+            SimplePie("average_claims") {
+                factionService.factions
+                    .map {
+                        claimService.getClaims(it.id).size
+                    }
+                    .average().toInt().toString()
+            }
+        )
+        metrics.addCustomChart(
+            SimplePie("total_claims") {
+                factionService.factions.sumOf {
+                    claimService.getClaims(it.id).size
```

Comment:
Again maybe worth doing on the DB? How does it perform with large numbers of factions and claims?

Reply:
Excellent question, no idea. My test server seems to start up pretty fast with the dev-generated test data though.